templates/web/bromley/footer.html

```diff
@@ -1,49 +1,44 @@
+                    [% IF pagefooter %]
```

Comment:
This footer ends up identical to the fixmystreet one - I don't think it needs adding in ba8ffd8